### DISCUSSION TITLE
Template (like issue templates) for PR

### DISCUSSION THREAD

**Discussion by Borda** [community/community#42973] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I am very happy with the recent feature - define issue templates with YAML which enforces required fields and clearly separates the requested user's input and some additional information. :+1: 
It would be great to allow the same also PR templates which are so far only plain text (with MD syntax), is there any consideration about bringing some more order in PR creation? :rabbit: 

Thank you, and looking forward to hearing more... :otter: 
Note that almost the same is already in PR topic: https://github.com/orgs/community/discussions/27897

**Comment by flying-sausages** [comment]
Functionally, how different are Issue templates from pull-request templates?
